File: lib/models/article.dart
class Articles {
  final int articleId;
  final int topicId;
  final String writerId;
  final String imageUrl;
  final String content;
  final DateTime createdAt;
  final DateTime updatedAt;

  Articles({
    required this.articleId,
    required this.topicId,
    required this.writerId,
    required this.imageUrl,
    required this.content,
    required this.createdAt,
    required this.updatedAt,
  });

  factory Articles.fromJson(Map<String, dynamic> json) {
    return Articles(
      articleId: json['articleId'],
      topicId: json['topicId'],
      writerId: json['writerId'],
      imageUrl: json['imageUrl'],
      content: json['content'],
      createdAt: DateTime.parse(json['createdAt']),
      updatedAt: DateTime.parse(json['updatedAt']),
    );
  }
}

File: dummy_backend/models/Article.js
const { DataTypes } = require('sequelize');
const sequelize = require('../sequelize');

const Article = sequelize.define('Article', {
  articleId: {
    type: DataTypes.INTEGER,
    primaryKey: true,
    autoIncrement: true,
  },
  topicId: {
    type: DataTypes.INTEGER,
    allowNull: false,
  },
  writerId: {
    type: DataTypes.STRING,
    allowNull: false,
  },
  imageUrl: {
    type: DataTypes.STRING,
    allowNull: false,
  },
  content: {
    type: DataTypes.TEXT,
    allowNull: false,
  },
  createdAt: {
    type: DataTypes.DATE,
    allowNull: false,
    defaultValue: DataTypes.NOW,
  },
  updatedAt: {
    type: DataTypes.DATE,
    allowNull: false,
    defaultValue: DataTypes.NOW,
  },
});

module.exports = Article;
